Maple2.Server.Game/PacketHandlers/TutorialItemHandler.cs

```diff
@@ -0,0 +1,50 @@
+using System.Linq;
+using Maple2.Model;
+using Maple2.Model.Enum;
+using Maple2.Model.Game;
+using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.PacketHandlers;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.PacketHandlers;
+
+public class TutorialItemHandler : PacketHandler<GameSession>
+{
+    public override RecvOp OpCode => RecvOp.RequestTutorialItem;
+
+    public override void Handle(GameSession session, IByteReader packet)
+    {
+        JobTable jobTable = session.TableMetadata.JobTable;
+
+        JobCode jobCode = session.Player.Value.Character.Job.Code();
+
+        if (!jobTable.Entries.TryGetValue(jobCode, out JobTable.Entry? jobMetadata))
+        {
+            return;
+        }
+
+        foreach (JobTable.Item tutorialItem in jobMetadata.Tutorial.StartItem)
```

Comment:
I don't like that for classes like thief we are still running this loop twice even though the first iteration already rewards all start items. How about doing a `GroupBy` on `StartItem` by `Id` so that we can loop over just unique ids. For each group you can sum up the count to get the total items needed.

Reply:
Could you check what I did for this? I'm not sure it's what you're expecting 😅